frontend/package.json

```diff
@@ -14,36 +15,39 @@
     "license-checker-windows": "cd scripts && windows_check_licenses.bat"
   },
   "dependencies": {
-    "@hookform/resolvers": "^3.3.1",
+    "@hookform/resolvers": "^3.3.4",
     "@vitejs/plugin-react-refresh": "^1.3.6",
-    "autoprefixer": "^10.4.15",
+    "autoprefixer": "^10.4.17",
+    "keycloak-js": "^23.0.6",
+    "postcss": "^8.4.35",
```

Comment:
What are We using this for?

Reply:
postcss is the tailwindcss dependency.
But good catch. I was experimenting with postcss configuration (I was trying to migrate it from .js to .ts, but it turns out it is impossible yet) and forgot to remove it. Although it is a dependency for tailwindcss we do not need to have it explicitly installed in our project, we just need to have a config file for it (postcss.config.js). I removed it from our dependencies.